### ISSUE TITLE
Linter - Refers to line numbers that don't exist

### ISSUE THREAD

**Issue by Deekers** [hebaishi/linter-gcc#42]
I have linter 1.11.4 and linter-gcc 0.6.8 on Atom 1.6.0.  Also, here is the output from gcc --version:

Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/c++/4.2.1
Apple LLVM version 7.0.2 (clang-700.1.81)
Target: x86_64-apple-darwin15.2.0
Thread model: posix

I am new to Atom/PlatformIO but have got everything basically working in that I can compile projects, upload Arduinos, etc with no problems.  But I am having problems with lint.  This is what I see:

#1 Lint reports issues to line numbers that do not exist.   These, of course, are all beyond the last line in the file.

#2 Lint is reporting errors that are flagged at lines that have no code or can't possibly have the errors indicated. 

There are no messages from the console, I have tried various installs and updates, but to no avail. 

The attached png shows lint pointing into the middle of some comments delineated by /* */
<img width="1440" alt="screen shot 2016-03-25 at 8 05 59 am" src="https://cloud.githubusercontent.com/assets/6326148/14089037/9d00ff26-f4e7-11e5-993b-4ee3ff2466f7.png">

**Comment by hebaishi** [comment]
Could you perform the following steps:
1. In the linter-gcc configuration settings, enable debugging messages
2. Press ```Ctrl+Shift+I```, copy the linter-gcc command and paste it here.
3. Paste the linter-gcc command into a terminal, and write the output here.

**Comment by Deekers** [comment]
Hi,
    I completed what you requested and I see that lint is doing its' job but the
lint "bubbles" on the panels are reporting the wrong file names.  This would
certainly explain why there is a mismatch in line numbers and reporting of
errors/problems that shouldn't exist.

   Is this still your issue or should I repost this to a different group now?

   I can still send the output if you want.

Thanks, Kent

**Comment by hebaishi** [comment]
No that's fine. I am planning to publish an update that should fix the filename issue. Let's keep this issue open until the problem is resolved.

**Comment by hebaishi** [comment]
Please update to version 0.6.9 and let me know when you have everything working normally again.

**Comment by Deekers** [comment]
Early checks are looking good!  I will keep you posted after a few more days.

Thank you for all of your help and efforts.

**Comment by hebaishi** [comment]
Hello. Is everything working as expected now?

**Comment by Deekers** [comment]
Hi,
    Yes I think so.  At this point I would consider this a resolved problem.
Thank you for you help and quick response.
